README.md

```diff
@@ -1,37 +1,36 @@
 # Terratest
 
+[![Go Report Card](https://goreportcard.com/badge/github.com/gruntwork-io/terratest)](https://goreportcard.com/report/github.com/gruntwork-io/terratest)
+
 Terratest is a Go library that makes it easier to write automated tests for your infrastructure code. It provides a
 variety of helper functions and patterns for common infrastructure testing tasks, including:
 
-* Testing Terraform code
-* Testing Packer templates
-* Testing Docker images
-* Executing commands on servers over SSH
-* Working with AWS APIs
-* Making HTTP requests
-* Running shell commands
-* And much more
+- Testing Terraform code
+- Testing Packer templates
+- Testing Docker images
+- Executing commands on servers over SSH
+- Working with AWS APIs
+- Making HTTP requests
+- Running shell commands
+- And much more
 
 Terratest was developed at [Gruntwork](https://gruntwork.io/) to help maintain the [Infrastructure as Code
 Library](https://gruntwork.io/infrastructure-as-code-library/), which contains over 250,000 lines of code written
 in Terraform, Go, Python, and Bash, and is used in production by hundreds of companies. See [Open sourcing Terratest: a
 swiss army knife for testing infrastructure code](https://blog.gruntwork.io/open-sourcing-terratest-a-swiss-army-knife-for-testing-infrastructure-code-5d883336fcd5)
 for more info.
 
-
-
-
```

Comment:
What tool is reformatting the Markdown files? I actually prefer having more spacing between sections, as it make sit easier to visually see the structure of the file when looking at the raw Markdown text rather than the HTML output.

Reply:
I'm using https://github.com/prettier/prettier to automatically format the Markdown. If you don't like it, I can revert it.

Reply:
I'm not religious about these sorts of decisions, but which of these makes the structure of the Markdown easier to see?

https://raw.githubusercontent.com/mastertinner/terratest/813222c9ce7fbdf4320e5b596f006602c98078c4/README.md
https://raw.githubusercontent.com/gruntwork-io/terratest/master/README.md

Reply:
I usually try to avoid having to maintain the formatting of my files manually because you'll almost always forget about doing it in one or more places and there's no way of enforcing/checking that it's actually done consistently. That's why I love tools like prettier because they automatically take care of it for me upon saving a file.

I'll leave the choice up to you. I agree that the file is visually easier to scan with the spacing you had before I auto-formatted it so if that's what you prefer, I'll gladly change it back.

Reply:
Totally agree on using automatic code formatting where possible, but it's important that the resulting formatting is easy to read. I've found prettier often favors consistency over readability, which is not always a good trade-off. If there's a way to configure prettier to enforce better spacing in Markdown, I'm all for it. If not, I'd prefer to keep the file as it was before. Thx!

Reply:
I put the spacing back.